server/views/pages/submit-application/swap-visiting-orders-for-pin-credit.njk

```diff
@@ -1,18 +1,34 @@
-{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
-{% from "govuk/components/button/macro.njk" import govukButton %}
-{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
-
 {% extends "../../partials/layout.njk" %}
 
 {% set pageTitle = applicationName + " - " + title %}
 {% set mainClasses = "app-container govuk-body applications-landing-page" %}
 
 {% block content %}
   <div class="govuk-body govuk-width-container">
-    {{ govukBackLink({
-      text: "Back",
-      href: "/"
-    }) }}
-
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <div class="govuk-panel govuk-panel--confirmation">
```

Comment:
Should be using the nunjucks components rather than the HTML, https://design-system.service.gov.uk/components/panel/